src/lib/y2users/group.rb

```diff
@@ -82,7 +82,16 @@ def initialize(name)
     def users
       return [] unless attached?
 
-      config.users.select { |u| u.gid == gid || users_name.include?(u.name) }
+      config.users.select { |u| same_gid?(u) || users_name.include?(u.name) }
+    end
+
+  private
+
+    # @see #users
```

Comment:
np: I would maybe just explain why two nils are not same gid ( group nil is not yet assigned and user nil means do not change )